### PR TITLE
Handle pydantic v1 and v2

### PR DESCRIPTION
**Pull request by PhilThev**
- Fix some breaking changes if we use pydantic v2 instead of v1
 - Bump python-on-whales version to 0.64.2 (currently the latest)

### CODE REVIEW

**Review by gabrieldemarmiesse — CHANGES_REQUESTED** [review]
Can you set an upper limit for pydantic in the requirements? <3 would be good

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
 python-dxf==7.8.0
 tqdm
 typer
-pydantic
+pydantic>=1.5,<3,!=2.0.*
```
Comment (gabrieldemarmiesse) [inline requirements.txt]:
```suggestion
pydantic>=1.5,<3
```
The part `!=2.0.*` doesn't come from docker-charon, it comes from python-on-whales. It's the responsibility of python-on-whales to impose this restriction.

**Review by gabrieldemarmiesse — CHANGES_REQUESTED** [review]

**Review by gabrieldemarmiesse — CHANGES_REQUESTED** [review]
Can you duplicate the CI so that the tests run with pydantic v1 and v2?

File: requirements.txt (hunk 1 above)
Comment (PhilThev) [reply requirements.txt]: Ok I didn't notice the restriction was about python-on-whales only, done !

**Review by gabrieldemarmiesse — APPROVED** [review]

**Pull request merged**